include/rocksdb/db.h

```diff
@@ -1443,6 +1443,11 @@ class DB {
       const ExportImportFilesMetaData& metadata,
       ColumnFamilyHandle** handle) = 0;
 
+  // Verify the checksums of files in db. Currently only tables are checked.
```

Comment:
It's not clear how this is different from VerifyChecksum. Please make that clear.

Reply:
Done